### PR TITLE
Swift tests: Stats

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: ablySpec/RestClient.stats.swift [hunk 1]
```diff
@@ -15,16 +15,14 @@ import Foundation
 private func postTestStats(stats: JSON) -> ARTClientOptions {
     let options = AblyTests.setupOptions(AblyTests.jsonRestOptions);
     
-    let key = ("\(options.key)" as NSString)
-        .dataUsingEncoding(NSUTF8StringEncoding)!
-        .base64EncodedStringWithOptions(NSDataBase64EncodingOptions(rawValue: 0))
+    let keyBase64 = encodeBase64(options.key ?? "")
     
-    let request = NSMutableURLRequest(URL: NSURL(string: "https://rest.ably.io/stats")!)
+    let request = NSMutableURLRequest(URL: NSURL(string: "https://sandbox-rest.ably.io/stats")!)
```
Comment (ricardopereira) [inline ablySpec/RestClient.stats.swift]: @mattheworiordan Is this correct? Prior I was getting 404.

Comment (mattheworiordan) [inline ablySpec/RestClient.stats.swift]: Yes, but surely the environment shouldn't be hard coded in and instead we should get the domain based on the environment you are testing against?

**Comment by ricardopereira** [comment]
@mattheworiordan I think it's all passing now. Swift and ObjC tests. Travis will tell if I am wrong.

**Comment by mattheworiordan** [comment]
> @mattheworiordan I think it's all passing now. Swift and ObjC tests. Travis will tell if I am wrong.

I look forward to that green :green_apple:

**Comment by mattheworiordan** [comment]
No :green_heart: unfortunately...

**Pull request closed without merging**